src/styles/blocks/card.scss

```diff
@@ -0,0 +1,131 @@
+.cards {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  width: 1024px;
+  margin: 0 auto;
+  padding: 50px 40px;
+}
+
+.card {
+  justify-self: center;
+  display: flex;
```

Comment:
Pay attention to your CSS properties order. Check yourself everywhere
https://9elements.com/css-rule-order/